globus_automate_client/flows_client.py

```diff
@@ -675,73 +753,91 @@ def enumerate_actions(
             params["orderby"] = ",".join(builder)
         return self.get(f"/runs", params=params, **kwargs)
 
+    def enumerate_actions(
+        self,
+        **kwargs,
+    ) -> GlobusHTTPResponse:
+        """
+        An alias for ``enumerate_runs``
+        """
+        return self.enumerate_runs(**kwargs)
+
     def list_flow_actions(
         self,
-        flow_id: str,
+        **kwargs,
+    ) -> GlobusHTTPResponse:
+        """
+        An alias for ``list_flow_runs``
+        """
+        return self.list_flow_runs(**kwargs)
+
+    def list_flow_runs(
+        self,
+        flow_id: Optional[str] = None,
         flow_scope: Optional[str] = None,
-        statuses: Optional[List[str]] = None,
-        roles: Optional[List[str]] = None,
+        statuses: Optional[Iterable[str]] = None,
+        roles: Optional[Iterable[str]] = None,
         marker: Optional[str] = None,
         per_page: Optional[int] = None,
         filters: Optional[dict] = None,
         orderings: Optional[dict] = None,
+        role: Optional[str] = None,
         **kwargs,
     ) -> GlobusHTTPResponse:
-        """
-        List all Actions that were launched as part of a Flow's run
+        """List all Runs for a particular Flow. If no flow_id is provided, all runs for all
+        Flows will be returned.
 
-        :param flow_id: The UUID identifying the Flow which launched the Action
+        :param flow_id: The UUID identifying the Flow which launched the Run. If not
+            provided, all runs will be returned regardless of which Flow was used to start
+            the Run (equivalent to ``enumerate_runs``).
 
         :param flow_scope: The scope associated with the Flow ``flow_id``. If
             not provided, the SDK will attempt to perform an introspection on
             the Flow to determine its scope automatically
 
-        :param statuses: A list of statuses used to filter the Actions that are
-            returned by the listing. Returned Actions are guaranteed to have one
-            of the specified ``statuses``. Valid values are:
+        :param  statuses: The same as in ``enumerate_runs``.
```

Comment:
Is there a way to make the `enumerate_runs`text show up as a link to the relevant docs? That should make it easier for someone trying to get a handle on this signature.

Reply:
That's a good suggestion. I don't now know of any good sphinx-y trick to do that. Rudy's comment the other day about writing something to pull the docstring from one function and use it to create the docstring of another might be a different solution. We probably end up with more verbose docs as there'd be replication, but maybe that's better than an open-ended sort of reference?

Again, I'm open to suggestions on how to improve something I know is non-ideal.